lib/src/action.dart

```diff
@@ -20,11 +20,22 @@ import 'package:w_common/disposable.dart';
 
 import 'package:w_flux/src/constants.dart' show v3Deprecation;
 
+/// Like [ActionV2], but payloads cannot be made non-nullable since the argument
+/// to [call] is optional.
+@Deprecated('Use ActionV2 instead, which supports non-nullable payloads.')
+class Action<T> extends ActionV2<T> {
```

Comment:
Now that ActionV2 is the common base class, I think we will also need to update the typing in the `store.dart` file for any parameter that currently expects type `Action` to expect type `ActionV2` instead - that way they will be able to accept either an `Action` or an `ActionV2` as people migrate.